doc/reference/filters.md

```diff
@@ -0,0 +1,60 @@
+# Instance filtering
+
+Operations Center uses [expr-lang](https://expr-lang.org/) for filtering in
+various places, including filtering of results fetched from the inventory
+as well as selection of updates and update files.
+
+See [Inventory](inventory) for more information about filtering inventory
+results.
+See [Update](update) for more information about update configuration.
+
+Expr is an expression language that uses a Go-like syntax with some more
+human-readable operators.
+
+```{note}
+For a full list of available functions and examples for more advanced expressions,
+see the full language definition at https://expr-lang.org/docs/language-definition.
+```
+
+## Common examples
+
+### Filtering results from inventory
+
+| Expression                                          | Entity  | Description                                                                          |
+| :---                                                | :---    | :---                                                                                 |
+| `name != "incusbr0" and name != "meshbr0"`          | network | Get all network except the automatically created ones named `incusbr0` and `meshbr0` |
```

Comment:
The filters in this document assume https://github.com/FuturFusion/operations-center/pull/375 to be merged, such that all the objects and properties available in expr are snake case (as in the JSON tags).